src/main/webapp/partials/instructorHelpCourses.jsp

```diff
@@ -821,6 +833,282 @@
       </div>
     </div>
   </div>
+  <h3>Restoring Courses</h3>
+  <div class="panel-group">
+    <div class="panel panel-default" id="course-view-deleted">
+      <div class="panel-heading cursor-pointer" data-toggle="collapse" data-target="#course-view-deleted-body">
+        <h3 class="panel-title">How do I view courses I have deleted?</h3>
+      </div>
+      <div id="course-view-deleted-body" class="panel-collapse collapse">
+        <div class="panel-body">
+          <p>
+            You can view all your deleted courses by navigating to the <b>Courses</b> page.<br>
+            Scroll to the <b>Deleted Courses</b> heading, which looks similar to this:
```

Comment:
Minor nit, but the section is actually `Deleted courses`.....

Reply:
Uh... didn't notice this. Then maybe the "Archived Courses" above also need to change :)